ucrop/src/main/java/com/yalantis/ucrop/task/BitmapLoadTask.java

```diff
@@ -168,7 +171,7 @@ private void processInputUri() throws NullPointerException, IOException {
                 throw e;
             }
         } else if ("content".equals(inputUriScheme)) {
-            String path = FileUtils.getPath(mContext, mInputUri);
+            String path = getFilePath();
```

Comment:
I think it's better to check for permission in here. And in case there is no one, then throw exception.
Something like throw new IllegalStateException("No permission for storage was provided"); Bacause now if just null returned, than user of the lib will get NoSuchFileEcxeption and it's misleading.